client/src/project/Project.present.js

```diff
@@ -263,6 +263,70 @@ function ProjectViewHeaderOverviewDescription({ settingsReadOnly, description, s
   return null;
 }
 
+const ProjectSuggestActions = (props) => {
+  const [ countCommitsReadme, setCountCommitsReadme ] = useState(null);
+  const commits = props.projectCoordinator.get("commits");
+  const countTotalCommits = commits?.list?.length ?? 0;
+  const hasDatasets = props.core?.datasets?.length > 0;
+  const isLoadingData = !commits.fetched ||
+    isRequestPending(props, "commitsReadme") ||
+    countCommitsReadme === null ||
+    typeof (props.core?.datasets) === "string";
+
+  useEffect(() => {
+    props.fetchDatasets();
+  }, []); // eslint-disable-line
+
+  useEffect(() => {
+    async function getReadmeCommits () {
+      const request = await props.projectCoordinator.fetchReadmeCommits();
+      setCountCommitsReadme(request);
+    }
+    if (props.projectCoordinator?.get("metadata.id") !== null)
+      getReadmeCommits();
+  }, [countTotalCommits]); // eslint-disable-line
+
+  if (isLoadingData || countTotalCommits > 4 || (countCommitsReadme > 1 && hasDatasets)) return "";
```

Comment:
I think it would be better to return null here. There is no need to add an empty string to the DOM.

```suggestion
  if (isLoadingData || countTotalCommits > 4 || (countCommitsReadme > 1 && hasDatasets)) return null;
```

Reply:
sure, updated to return null